integreat_cms/cms/templates/socialmedia_headers.html

```diff
@@ -0,0 +1,13 @@
+<title>{{ title }}</title>
```

Comment:
```suggestion
<html lang="{{ language_code }}">
<title>{{ title }}</title>
```
What do you think about the idea to include the `<html>` tag here as well to make it possible for the bots to detect the content language of the page?

Reply:
But we could put the canonical urls inside here https://github.com/digitalfabrik/integreat-app/blob/4dd43127ffae65ae1326640e67b5e6ab22ec2bc1/web/src/components/Helmet.tsx#L24

Reply:
The way I understood nginx server side includes is that they include arbitrary content into the final html document before the response is sent to the client, right? So if nginx doesn't know how valid HTML looks like, we could also include a partial document, what could possibly go wrong? :see_no_evil: 

Change [index.ejs#L2-L17](https://github.com/digitalfabrik/integreat-app/blob/4dd43127ffae65ae1326640e67b5e6ab22ec2bc1/web/src/index.ejs#L2-L17) to:
```
<!-- Nginx Server Side Include template for dynamic social media previews -->
<!--# if expr="$render_title = yes" -->
<!--# include virtual="/proxy/socialmeta/$request_uri" -->
<!--# else -->
<html lang="">
<head>
    <title><%= config.appName %></title>
    <meta name="apple-mobile-web-app-title" content="<%= config.appName %>">
    <meta property="og:title" content="<%= config.appName %>" data-react-helmet="true">
    <meta property="og:site_name" content="<%= config.hostName %>">
    <meta name="twitter:title" content="<%= config.appName %>">
<!--# endif -->
```

And then on the cms side, we only render the opening tag of the `head` without closing it:
```
<html lang="{{ language_code }}">
<head>
    <title>{{ title }}</title>
    ...
```

Reply:
Hmm. then we'd have to persuade @svenseeberg to always serve the socialmediaheaders - independently of the User-Agent. Otherwise we'd serve invalid html 😅 I'm not yet persuaded...

Reply:
Well, but that's what the `<!--# if expr="$render_title = yes" -->` is for, right?
So if the value is set, the dynamic social media headers are included, and if not, we use the hardcoded values from the `<!--# else -->` case... so we'd serve valid HTML in any case :nerd_face:

Reply:
Ah right. Me not `ejs`-Master 😅 
Thanks for explaining.